unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/ChatNotificationController.cs

```diff
@@ -124,19 +127,25 @@ private async UniTaskVoid WaitThenFadeOutNotifications(CancellationToken cancell
 
             mainChatNotificationView.HideNotifications();
 
-            if (topNotificationPanelTransform.Get().gameObject.activeInHierarchy)
+            if (topNotificationPanelTransform.Get() != null && topNotificationPanelTransform.Get().gameObject.activeInHierarchy)
                 topNotificationView.HideNotification();
         }
 
         private string ExtractPeerId(ChatMessage message) =>
             message.sender != ownUserProfile.userId ? message.sender : message.recipient;
 
+        private void ResetVisibility(bool current, bool previous){
```

Comment:
format issues, `{}` on new lines. You can do an inliner though: `private void ResetVisibility(bool current, bool previous) => SetVisibility(current);`

Reply:
good point, my bad, fixed